text_adventure.py

```diff
@@ -0,0 +1,190 @@
+import random
+
+class GameObject:
+    verb_dict = {
+        "examine": "Examine a creature in the room.",
+        "hit": "Hit a creature in the room.",
+        "help": "Get help about the available commands.",
+        "heal": "Heal for an amount of damage.",
+        "kill": "Immediately kill the target.",
+        "status": "Get your own status."
+    }
+
+    @staticmethod
+    def get_input():
+        command = input(": ").split()
+        verb_word = command[0]
+        if verb_word in GameObject.verb_dict:
+            verb = getattr(GameObject, verb_word)
```

Comment:
fascinating approach! what you've done here is a form of dynamic dispatch. I see that you're checking that the verb_word is a legal one in the previous line so that's good, but this sort of approach can lead to security holes (if you get rid of the in check for perf reasons, then someone could pass in get_input as the verb word and weird things happen.